mosquitto/Dockerfile

```diff
@@ -1,16 +1,16 @@
-ARG BUILD_FROM
+ARG BUILD_FROM=ghcr.io/home-assistant/amd64-base:3.13
```

Comment:
should be set from `build.json`

Reply:
It is, this is just for local building without passing anything. Will remove.